main.py

```diff
@@ -159,74 +156,56 @@ def __main__():
         # module. This combination allows one to run and debug their stix-shifter code locally, while interacting with
         # it inside a service provider such as IBM Security Connect
         app = Flask("stix-shifter")
-        
+
         @app.route('/transform_query', methods=['POST'])
         def transform_query():
-            request_args = request.get_json( force=True )
-            translation = stix_translation.StixTranslation()
-            dsl = translation.translate(args.translation_module, 'query', args.data_source, request_args["query"])
-            return json.dumps(dsl)
+            print("transforming query")
+            host = ProxyHost()
+            return host.transform_query(host)
```

Comment:
do we still need `host` object to pass in as argument?

Reply:
No, looks like I missed that one.